SDK.markdown

```diff
@@ -177,6 +177,83 @@ client.auth.cancelMFAConfirmation();
 
 The Promise returned by `getMFAConfirmedToken` will then reject with a `MFAConfirmationCanceledError`.
 
+
+### Device Binding
+To have access to Kontist API endpoints that require strong customer authentication, you need to pass multi-factor authentication (MFA).
+
+To make it seamless for you we provide device binding authentication that leverages digital signature algorithm.
+
+To pass device binding authentication you need to generate a pair of private and public keys by using elliptic curve algorithm (secp256r1). To create and verify your device you need to pass your public key and then sign OTP from SMS received on your phone.
+
+After that, you can use your device id and your private key to get your confirmed auth token by creating and verifying device challenge.
+
+#### Create device
+To initiate the device binding you need to create a device by providing your **public key**.
+
+Please, make sure that you **never share your private key** and it's **stored in a secure place** (do not store it in a browser).
+
+After a successful request, you will receive an SMS with OTP that you will need during device verification.
+
+```typescript
+const result = await client.auth.createDevice({
+  name: "iPhone XS",
+  key: "..." // The hex-encoded public key without header
+});
+```
+
+`result` then has following structure:
+
+```javascript
+{
+  deviceId: "daecde61-18a4-4010-a0f7-a8b21c27996a",
+  challengeId: "daecde61-18a4-4010-a0f7-a8b21c27996a"
+}
+```
+
+#### Verify device
+To verify device you need to provide a signature of OTP received on your mobile phone.
+
```

Comment:
As discussed it makes sense to add a couple of words here how this `signature` has to be created from the private key.